crates/solvers/src/infra/dex/balancer/dto.rs

```diff
@@ -9,7 +9,7 @@ use {
 };
 
 #[derive(Debug, Serialize)]
-#[serde(rename_all = "lowercase")]
```

Comment:
I believe this API is also maintained by balancer and thus may break (but given the non compound words, it may not be an issue here)